<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Regular</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CPU:1, Memory: 128MB, Size: 5GB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Regular</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CPU:1, Memory: 256MB, Size: 10GB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Regular</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CPU:1, Memory: 512MB, Size: 20GB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name </string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_btn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Regular</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SUBMIT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Root Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_input">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="pw_input">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_btn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Regular</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CANCEL</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="type1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>108</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Type 1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="type2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>108</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Type 2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="type3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>108</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Regular</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Type 3</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDemo">
    <property name="title">
     <string>Demo</string>
    </property>
   </widget>
   <addaction name="menuDemo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
